clarity_epp/export/caliper.py

```diff
@@ -64,7 +64,7 @@ def samplesheet_normalise(lims, process_id, output_file):
     volume_DNA = {}
     volume_H2O = {}
     conc_measured = {}
-    output_ng = process.udf['Output genormaliseerd gDNA']
+    output_ng = float(process.udf['Output genormaliseerd gDNA'])
```

Comment:
output_ng is used in a calculation on line 137, by making it a float this calculation will also result in a float and not in rounded number (int).

https://github.com/UMCUGenetics/clarity_epp/blob/923593326bb78d569f2f6530301a977cf8d85a4a/clarity_epp/export/caliper.py#L137

Reply:
1. You are right, removed them.
2. A udf stands for 'user defined field', you can configure these within the clarity lims system. This one is configured to be a number, this is tested during validations. The script assumes that configuration is correct.

Reply:
then I am confident it works the way it does. thanks for the explanation